### PR TITLE
DNM *(ticdc): Embed pd info into changefeed

### LINKED ISSUE #5301: support multi tidb cluster in one ticdc cluster
PRs: 
- [x] https://github.com/pingcap/tiflow/pull/5300
- [x] https://github.com/pingcap/tiflow/pull/5282
- [x] https://github.com/pingcap/tiflow/pull/5311

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #5301

### What is changed and how it works?
embed pd information into changefeed

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/model/changefeed.go [hunk 1]
```diff
@@ -137,6 +137,12 @@ type ChangeFeedInfo struct {
 	SyncPointEnabled  bool          `json:"sync-point-enabled"`
 	SyncPointInterval time.Duration `json:"sync-point-interval"`
 	CreatorVersion    string        `json:"creator-version"`
+
+	PDEndpoints   string   `json:"pd-endpoints"`
+	KeyPath       string   `json:"key-path"`
+	CertPath      string   `json:"cert-path"`
+	CAPath        string   `json:"ca-path"`
```
Comment (overvenus) [inline cdc/model/changefeed.go:144]: By embedding local path in changefeed, we may face the same problem as sort-dir in changefeed, not every TiCDC node has the same file system topology.

Comment (overvenus) [inline cdc/model/changefeed.go:144]: What about storing cluster info in another struct and embed a pointer to that struct? Pros: When cluster info changes, we only need to change the struct not all changefeeds.

**Comment by sdojjy** [comment]
store cluster-info in another etcd key

**Pull request closed without merging**